src/summarizedexperiment/utils.py

```diff
@@ -0,0 +1,80 @@
+from typing import Optional, Sequence, MutableMapping
+from functools import reduce
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+
+from .SummarizedExperiment import SummarizedExperiment
+
+
+def concat(
+    ses: Sequence[SummarizedExperiment],
+    how: str = "outer",
+    metadata: Optional[MutableMapping] = None,
+) -> SummarizedExperiment:
```

Comment:
page 26 from [Bioc's SE manual](https://bioconductor.org/packages/devel/bioc/manuals/SummarizedExperiment/man/SummarizedExperiment.pdf) describes these operations. 

I think we want to clarify the intent of this action. 

`cbind, rbind, combineRows, combineCols`